deployments/kubectl/configMap.yaml

```diff
@@ -18,6 +18,9 @@ data:
   RULES_CONFIGMAP_NAME: oathkeeper-rules
   RULES_CONFIGMAP_FILE_NAME: access-rules.json
   RULES_CONFIGMAP_NAMESPACE: default
+  OPENFGA_API_SCHEME: http
+  OPENFGA_API_HOST: openfga.default.svc.cluster.local:8080
+  OPENFGA_API_TOKEN: "42"
```

Comment:
Should we add: 
```
AUTHORIZATION_ENABLED: "false"
```

To make editing the configmap easier after openfga setup is complete?